animations-dbus/animations-dbus-server-object.c

```diff
@@ -454,7 +456,7 @@ attempt_to_own_session_bus_name (GDBusConnection *connection,
                                        on_got_session_bus_name,
                                        on_lost_session_bus_name,
                                        task,
-                                       g_object_unref);
+                                       NULL);
```

Comment:
I think this will introduce some new bugs: `g_dbus_own_name_on_connection()` is allowed to call the name-acquired and name-lost callbacks multiple times, so dropping the reference to the `GTask` after one of them completes means there will be double- and triple-unrefs in future. I think the original code here is correct, and [the bug](https://github.com/endlessm/eos-hack-extension/pull/16#discussion_r410179888) is somewhere else.

Reply:
(Outdated.)